pyscript.core/tests/integration/support.py

```diff
@@ -126,7 +126,7 @@ def filter_page_content(lines, exclude=None):
 
 
 @pytest.mark.usefixtures("init")
-@with_execution_thread("main", "worker")
+@with_execution_thread("main") # , "worker") # XXX re-enable workers eventually
```

Comment:
This is essentially waiting on a re-write of how the `execution_thread` interacts with the actual test setup yeah? Not a blocker, just curious.

Reply:
yes, I have a WIP branch for that, but I didn't do much progress because so far I haven't found a good way to detect when all the tags have finished execution, but hopefully it will come soon